src/Microsoft.AspNet.OData.Shared/TypelessPatchMethodHandler.cs

```diff
@@ -0,0 +1,196 @@
+﻿using Microsoft.OData.Edm;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.OData
+{    
+   
+    /// <summary>
+    /// Handler Class to handle users methods for create, delete and update
+    /// </summary>
+    public abstract class TypelessPatchMethodHandler 
```

Comment:
>TypelessPatchMethodHandler [](start = 26, length = 26)

the nameing is weird.

You can just use PatchMethodHandler (Be noted, without the <TStructuralType>),

Reply:
Just wanted to call it out seperately for typeless, to distinguish 

---
In reply to: [604618607](https://github.com/OData/WebApi/pull/2455#discussion_r604618607) [](ancestors = 604618607)

Reply:
Maybe IEdmPathMethodHandler, to follow the pattern for non clr-backed classes?

---
In reply to: [604693905](https://github.com/OData/WebApi/pull/2455#discussion_r604693905) [](ancestors = 604693905,604618607)